### PR TITLE
Add Document w/ Eng Sys Checks

### PR DESCRIPTION
**Pull request by scbedd**
@praveenkuttappan did this. I'm adding some additional detail, moving into the public, and taking care of any other housekeeping here.

### CODE REVIEW

File: doc/eng_sys_checks.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+ 
+•   Regression test 
+Variable name: `Run.Regression`
+Value: true 
+
+
+
+## Release Checks
+
+<will be updated soon>
```
Comment (praveenkuttappan) [inline doc/eng_sys_checks.md:200]: Should we leave it this for now. or keeping it as coming soon.

**Review by praveenkuttappan — DISMISSED** [review]
Looks perfect

File: doc/eng_sys_checks.md (hunk 1 above)
Comment (scbedd) [reply doc/eng_sys_checks.md]: Man I'm dumb. I actually thought I cleaned that up. *screaming*

**Pull request merged**